### PR TITLE
Add offset for times

### PR DESCRIPTION
**Pull request by nick-zh**
https://github.com/arnaud-lb/php-rdkafka/issues/130

### CODE REVIEW

**Comment by nick-zh** [comment]
@arnaud-lb if you find the time, i would appreciate if you can have a look and check if i did everything properly, since this is my first contribution. Thx in advance man :smile:

File: rdkafka.c [hunk 1]
```diff
@@ -589,6 +589,48 @@ PHP_METHOD(RdKafka__Kafka, queryWatermarkOffsets)
...
+    topicPartitions = array_arg_to_kafka_topic_partition_list(1, htopars TSRMLS_CC);
+    if (!topicPartitions) {
+        return;
+    }
+
+    err = rd_kafka_offsets_for_times(intern->rk, topicPartitions, timeout_ms);
+
+    if (err != RD_KAFKA_RESP_ERR_NO_ERROR) {
+        zend_throw_exception(ce_kafka_exception, rd_kafka_err2str(err), err TSRMLS_CC);
+        return;
```
Comment (arnaud-lb) [inline rdkafka.c:626]: We might need to destroy topicPartitions before return here

Comment (nick-zh) [reply rdkafka.c:626]: i added a destroy before throwing the exception, thanks for the feedback

**Comment by arnaud-lb** [comment]
Thanks !

**Pull request merged**